src/Microsoft.AspNetCore.Razor.Tasks/RazorTagHelper.cs

```diff
@@ -0,0 +1,47 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+
+using System.IO;
+using System.Text;
+using Microsoft.Build.Framework;
+
+namespace Microsoft.AspNetCore.Razor.Tasks
+{
+    public class RazorTagHelper : DotNetToolTask
+    {
+        [Required]
+        public string[] Assemblies { get; set; }
+
+        [Output]
+        [Required]
+        public string TagHelperManifest { get; set; }
+        
+        protected override bool SkipTaskExecution()
+        {
+            if (Assemblies.Length == 0)
+            {
+                // If for some reason there are no assemblies, we have nothing to do. Let's just
+                // skip running the tool.
+                File.WriteAllText(TagHelperManifest, "{ }");
```

Comment:
Is it even possible in practice for there to be 0 assemblies? I'd lean towards blowing up if we have 0

Reply:
It's more msbuild friendly to noop